apps/ejabberd/src/mod_muc_light_admin.erl

```diff
@@ -119,9 +119,10 @@ create_unique_room(Domain, RoomName, Creator, Subject) ->
 
 invite_to_room(Domain, RoomName, Sender, Recipient) ->
     R = muc_light_room_name_to_jid(jid:from_binary(Sender), RoomName, Domain),
-    S = jid:from_binary(Sender),
+    S = jid:replace_resource(jid:from_binary(Sender), <<>>),
```

Comment:
Yes, that's what we usually do when we want to check if there is no stanza for a user. There is `escalus_assert:has_no_stanzas/1` function which you can use.